lib/printer.rb

```diff
@@ -0,0 +1,14 @@
+class Printer
```

Comment:
This responsibility extraction is really good and helps make the code easier to change.